assembl/static2/js/app/pages/questionModeratePosts.jsx

```diff
@@ -0,0 +1,25 @@
+// @flow
+import React from 'react';
+
+import Question, { type Props } from './question';
+import { connectedUserIsAdmin } from '../utils/permissions';
+import { get, goTo } from '../utils/routeMap';
+
+class QuestionModeratePosts extends React.Component<Props> {
+  componentDidMount() {
+    if (!connectedUserIsAdmin()) {
```

Comment:
You should probably check for Permissions.MODERATE_POST permission instead of Permissions.ADMIN_DISCUSSION if you also change it for P_MODERATE in the ValidatePost mutation on the backend.